src/mono/mono/eventpipe/ep-rt-mono.h

```diff
@@ -1381,16 +1366,6 @@ ep_rt_utf8_string_replace (
 	return false;
 }
 
-static
-inline
-ep_char16_t *
-ep_rt_utf8_to_utf16le_string (
-	const ep_char8_t *str,
-	size_t len)
-{
-	return (ep_char16_t *)(g_utf8_to_utf16le ((const gchar *)str, (glong)len, NULL, NULL, NULL));
```

Comment:
Last two usages of `g_utf8_to_utf16le` are deleted in this PR, definition from `src/mono/mono/eglib/glib.h` and `src/mono/mono/eglib/giconv.c` can also be removed. (`g_utf8_to_utf16le_custom_alloc` can stay for `genEventPipe.py` generator)